docs/main/concepts/data-engines/data-engines.md

```diff
@@ -120,12 +120,12 @@ The below table identifies few differences among the different OpenEBS Local Eng
 | Works on OS Mounted storage                  |  Yes      |   Yes     | No         | No       | No       |
 
 
-### Replicated Engines
+### Replicated Storages
 
-Replicated Engines (aka Replicated Volumes) are those that can synchronously replicate the data to multiple nodes. These engines provide protection against node failures, by allowing the volume to be accessible from one of the other nodes where the data was replicated to. The replication can also be setup across availability zones helping applications move across availability zones. Replicated Volumes are also capable of enterprise storage features like snapshots, clone, volume expansion, and so forth. 
+Replicated Storages (aka Replicated Volumes) are those that can synchronously replicate the data to multiple nodes. These engines provide protection against node failures, by allowing the volume to be accessible from one of the other nodes where the data was replicated to. The replication can also be setup across availability zones helping applications move across availability zones. Replicated Volumes are also capable of enterprise storage features like snapshots, clone, volume expansion, and so forth. 
```

Comment:
change a.k.a Replicated Volumes to a.k.a Replicated Engines. Same thing needs to be done for Local as well.

Something like this:
Replicated Storage (a.k.a Replicated Engines)

Reply:
I have added terms for the first instance in the topic. Rest of the document references - Local Storage/Replicated Storage


### Local Storage

OpenEBS **Local Storage** or **Local Engines** can create Persistent Volumes (PVs) out of local disks or hostpaths or use the volume managers on the Kubernetes worker nodes. 

### Replicated Storage

Replicated Storage or Replicated Engines (f.k.a Mayastor) are those that can synchronously replicate the data to multiple nodes.